qa/tasks/cephfs/kernel_mount.py

```diff
@@ -202,18 +180,54 @@ def read_debug_file(self, filename):
                 return None
             raise
 
+    def _get_global_id(self):
+        try:
+            p = self.run_shell_payload(f"getfattr --only-values -n ceph.client_id .", stdout=StringIO())
```

Comment:
You're probably thinking of the python programs that call getxattr for us. Those are transmitted to the "client" host and then executed. It would probably be less efficient than calling `/bin/getfattr`.